### ISSUE TITLE
Question: Accessibility Features

### ISSUE THREAD

**Issue by porglezomp** [hecrj/iced#282]
Iced looks extremely nice in terms of visual quality, but I'm concerned about accessibility. I don't see any mention of it elsewhere, so I'm curious if there's any support for accessibility features currently implemented, and if there are future plans for explicitly supporting it? Thanks.

**Comment by hecrj** [comment]
>  I'm curious if there's any support for accessibility features currently implemented

There is no explicit support for accessibility features yet. This is missing, like many other features. 
The library is a few months old!

> and if there are future plans for explicitly supporting it

I think this is an important feature, and I agree we should at least mention it in the roadmap soon.

I believe the modular architecture could help us greatly here. The `iced` public API is meant to be high-level, and `iced_native` is shell-agnostic and renderer-agnostic. The same way we can target the DOM with `iced_web`, I think we should have enough room to target accessibility APIs, build screen readers, specific shells, etc.

**Comment by porglezomp** [comment]
Thanks for the answer! I wasn't trying to rush anything, just glad to see see that it's an issue being considered.

**Comment by Restioson** [comment]
Hopefully the default widgets can have automatic implementations, like with GTK/ATK (but cross platform).
